lib/css/modules/_redditUserInfo.scss

```diff
@@ -0,0 +1,5 @@
+.res-redditUserInfo-hideAuthorTooltip {
```

Comment:
This file must be imported [here](https://github.com/BenMcGarry/Reddit-Enhancement-Suite/blob/476b6dceb1eaaeb904470a4f323f3f02e3b916b7/lib/css/res.scss#L26)